styles.css

```
@@ -38,6 +38,64 @@ body {
   background-color: var(--surface-primary);
 }
 
+.form {
+  margin: 0 4rem;
+  margin-top: 4rem;
+}
+
+.form__paragraph {
+  color: var(--text-primary);
+  font-family: "Exo 2";
+  font-size: 1rem;
+  font-weight: 300;
+  line-height: 1.5rem;
+}
+
+.input {
+  margin-bottom: 1rem;
+  height: 2rem;
+  width: 100%;
+  max-width: 450px; /* just a fallback for large screen sizes */
+  border: 1px solid var(--details);
+  border-radius: 0.5rem;
+  padding: 0 1rem;
+  background-color: var(--surface-tertiary);
+  color: var(--text-tertiary);
+  font-family: "Exo 2";
+  font-weight: 300;
+  font-size: 1rem;
+  line-height: 1.5rem;
+}
+
+.textarea {
+  height: 10rem;
+  width: 100%;
+  max-width: 450px; /* just a fallback for large screen size */
+  max-height: 200px; /* just a fallback for large screen size */
+  border: 1px solid var(--details);
+  border-radius: 0.5rem;
+  padding: 0 1rem;
+  background-color: var(--surface-tertiary);
+  color: var(--text-tertiary);
+  font-family: "Exo 2";
+  font-weight: 300;
+  font-size: 1rem;
+  line-height: 1.5rem;
+}
+
+.button {
```

Seguindo o padrão da metodologia.

```suggestion
.form__submit-button {
```